### ISSUE TITLE
Problem with acr login in debian bullseye

### ISSUE THREAD

**Issue by josejuanmontiel** [Azure/azure-cli#19326]
## Describe the bug

**Command Name**
`az acr login`

**Errors:**
```
API version 2020-11-01-preview does not have operation group 'registries'
Traceback (most recent call last):
python3/dist-packages/knack/cli.py, ln 233, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 659, in execute
    raise ex
cli/core/commands/__init__.py, ln 722, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
cli/core/commands/__init__.py, ln 715, in _run_job
    six.reraise(*sys.exc_info())
...
cli/command_modules/acr/_client_factory.py, ln 20, in cf_acr_registries
    return get_acr_service_client(cli_ctx).registries
azure/mgmt/containerregistry/_container_registry_management_client.py, ln 304, in registries
    raise ValueError("API version {} does not have operation group 'registries'".format(api_version))
ValueError: API version 2020-11-01-preview does not have operation group 'registries'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az acr login --name {}`

## Environment Summary
```
Linux-5.10.0-7-amd64-x86_64-with-glibc2.31
Python 3.9.2
Installer: DEB

azure-cli 2.18.0

Extensions:
azure-devops 0.19.0

```

**Comment by yonzhan** [comment]
route to service team

**Comment by josejuanmontiel** [comment]
If downgrade the repo of microsoft to buster... work... but version in bullseye not

**Comment by aheld** [comment]
I had a similar issue w/ Pop_OS, it looks like the build for the latest ubuntu  `hirsute` isn't available. 

I manually edited the sources list to get the most recent that [Azure has](https://packages.microsoft.com/repos/azure-cli/ ), `focal` - and it seems to work so far. 

```
⚡  lsb_release -cs
hirsute
⚡  cat /etc/apt/sources.list.d/azure-cli.list 
deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ focal main
```

**Comment by jiasli** [comment]
@aheld, [Pop!_OS](https://pop.system76.com/) is not supported by Azure CLI. 

As long as it is a Ubuntu variation that follows the Ubuntu versioning:

```
$ lsb_release -is
Ubuntu
$ lsb_release -cs
focal/hirsute/...
```

it may work. However, you are basically on your own.

**Comment by aheld** [comment]
@jiasli - thanks!

pop os is basically hirsuite and `lsb_release -cs` does return hirsute.

when I checked https://packages.microsoft.com/repos/azure-cli/dists/ I didn't see hirsute so I imagine that ubuntu clients are using the maintainers version of azure-cli, and as per MS docs that one does not work.

when I manually updated the apt sources list to "downgrade" to focal, I reinstalled and it works fine now.

I'm good, just wanted to leave this here if others have similar issues.
